src/Leanplum.js

```diff
@@ -414,9 +426,13 @@ export default class Leanplum {
       info = undefined
     }
 
-    events.publish('track', {
-      event,
-      params
+    events.publish( 'track', {
+      messages: Leanplum.getFilteredResults(
```

Comment:
Why is the message sent to `track` channel, and not `track/message` here? Is there a difference in those channels?